src/main/java/org/maxgamer/quickshop/shop/ContainerShop.java

```diff
@@ -747,9 +801,10 @@ public void refresh() {
         }
         if (displayItem != null) {
             displayItem.remove();
-            initDisplayItem();
-            displayItem.spawn();
         }
+        initDisplayItem();
+        displayItem.spawn();
```

Comment:
Why we need to put two statements to the outside?

Reply:
Good point, in fact it's even better to not do that just in case it gets called on a left shop which shouldn't have an item. I thought I was making it more robust but that just opens up new possibilities for duplicates.,